CMakeLists.txt

```diff
@@ -23,6 +23,10 @@
 cmake_minimum_required(VERSION 3.8)
 project(units)
 
+# project options
+option(BUILD_TESTS "Build utils tests" ON)
+option(BUILD_EXAMPLES "Build utils examples" ON)
+
```

Comment:
This is not necessary as Conan should never export this CMake file. Only `src` directory is ever released to users and packaged with Conan (unless the user will directly specify `CONAN_RUN_TESTS`).

Exporting it would be a big mistake as this file modifies compiler warning flags. The only need for this file is to provide a development environment where tests and examples should always be built.